autoload.php

```diff
@@ -17,17 +17,28 @@
 
 if (file_exists('vendor/autoload.php')) {
     // prefer the composer autoloader
-    return require_once('vendor/autoload.php');
+    require_once('vendor/autoload.php');
+} elseif (!class_exists('TinCan\\Version')) {
```

Comment:
Don't cuddle else.

Is this check here just to make it work until #31 lands? I'm not sure I follow what you are doing here.